operator/pkg/validate/validate.go

```diff
@@ -54,10 +54,11 @@ func CheckIstioOperatorSpec(is *v1alpha1.IstioOperatorSpec, checkRequiredFields
 	}
 
 	errs = CheckValues(is.Values)
-	return util.AppendErrs(errs, validate(defaultValidations, is, nil, checkRequiredFields))
+	return util.AppendErrs(errs, Validate(DefaultValidations, is, nil, checkRequiredFields))
 }
 
-func validate(validations map[string]ValidatorFunc, structPtr interface{}, path util.Path, checkRequired bool) (errs util.Errors) {
+// Validate validates the values of the tree using the supplied Func
```

Comment:
may be add a note saying this is also used by third party integrations and that it shouldn't be made private?